.werft/post-process.sh

```diff
@@ -216,7 +218,10 @@ while [ "$i" -le "$DOCS" ]; do
     if [[ "ws-proxy" == "$NAME" ]] && [[ "$KIND" == "Service" ]]; then
       WORK="overrides for $NAME $KIND"
       echo "$WORK"
-      yq w -i k8s.yaml -d "$i" "metadata.annotations[cloud.google.com/neg]" '{"exposed_ports": {"22":{}}}'
+      # notice that current we use array index
+      yq w -i k8s.yaml -d "$i" "spec.ports.[0].port" 80
+      yq w -i k8s.yaml -d "$i" "spec.ports.[1].port" 443
+      yq w -i k8s.yaml -d "$i" "metadata.annotations[cloud.google.com/neg]" '{"exposed_ports": {"22":{},"80":{},"443":{}}}'
       yq w -i k8s.yaml -d "$i" spec.type LoadBalancer
```

Comment:
We'll need to think of a good strategy to make this work for most folks in a self-hosted setup. Not a concern for this PR, but a follow up.

Reply:
We're seeing folks who cannot use LoadBalancer at all.

In a follow up step we'll have to add a TCP tunnel to the meta caddy which serves as "ingress" in full installations. This way we can make the ssh gateway available in self-hosted installs without creating new requirements.

Reply:
I think Chris meant it as another PR. Let's file an issue for it and it to SSH proxy epic.